react-native-in-app-payments-quickstart/app/components/OrderInformationView.tsx

```diff
@@ -26,7 +26,7 @@ OrderInformationView.propTypes = {
   description: PropTypes.string.isRequired,
 };
 
-export default function OrderInformationView({ title, description }) {
+export default function OrderInformationView({ title }: { title: any }, { description }: { description: any }) {
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.